tools/crm_mon.c

```diff
@@ -1377,6 +1377,7 @@ main(int argc, char **argv)
         if (ncurses_winch_handler == SIG_DFL ||
             ncurses_winch_handler == SIG_IGN || ncurses_winch_handler == SIG_ERR)
             ncurses_winch_handler = NULL;
+        // cppcheck-suppress leakNoVarFunctionCall
         g_io_add_watch(g_io_channel_unix_new(STDIN_FILENO), G_IO_IN, detect_user_input, NULL);
```

Comment:
Looking at the docs, by default the file descriptor is not closed when the channel is destroyed, so maybe we should be freeing it. I can't imagine it mattering in crm_mon but if it's not too difficult it might be worthwhile